src/cljs/pyregence/components/map_controls/measure_tool.cljs

```diff
@@ -52,13 +52,21 @@
            [:p "Note: There is a +/- 0.5% Great-Circle calculation error."]]
           [lon-lat-position "Point One Location" (if @point-one @point-one [0 0])]
           [lon-lat-position "Point Two Location" (if @point-two @point-two [0 0])]
-          [:div {:style {:display         "flex"
-                         :flex-direction  "column"
-                         :margin-top      ".2rem"}}
-           (when (> @distance-between-points 0)
-             [:div {:style {:height 30 :margin ".2rem"}}
-              [:label {:style ($/padding "1px" :1)}
-               (str (cl-format nil "~,4f" @distance-between-points) " meters")]])
+          (when (> @distance-between-points 0)
+            [:strong {:style {:display         "flex"
+                              :justify-content "center"
+                              :margin          ".4rem"
+                              :font-size       "1.2rem"}}
+             (let [format (fn [number label] (str (cl-format nil "~,1f" number) " " label))
+                   meters  @distance-between-points
+                   miles (* 0.00062137 meters)
+                   kilometers (/ meters 1000)
+                   mi (format miles "miles")
+                   km (format kilometers "kilometers")]
```

Comment:
Another stylistic thing that we follow in Pyrecast: we align columns in `let` bindings like so:
```suggestion
             (let [format     (fn [number label] (str (cl-format nil "~,1f" number) " " label))
                   meters     @distance-between-points
                   miles      (* 0.00062137 meters)
                   kilometers (/ meters 1000)
                   mi         (format miles "miles")
                   km         (format kilometers "kilometers")]
```

Reply:
Also, it's not necessary, but if you wanted to add the meters to miles and meters to kilometers conversions to a util file, the right one would be `src/cljs/pyregence/utils/number_utils.cljs`.

Reply:
> We typically try to keep let bindings at the top-level. Can you move these to within the outer r/with-let?

Kind of, I don't think we can bind mi and km inside r/with-let as is done in the commit here because r/with-let only evals once, which would lock both of those in as 0. 

We can bind a function in the r/with-let which can take the meters/distance-between-points. That's what the latest [commit ](https://github.com/pyregence/pyregence/pull/890/commits/fbc48f3796264d8589ac2616dc0dbebf80b32c24) does, i'm not sure that's more clear.  I think either approach is fine though I prefer tighter scopes, less chance of confusion.